### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by PlanetNoobGotHacked**
yes

### CODE REVIEW

**Comment by PlanetNoobGotHacked** [comment]
i fixed a typo

**Comment by SkyBlockDev** [comment]
im glad you fixed a typo

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@
 ![GitHub License](https://img.shields.io/github/license/TeamClimactic/Rango?style=flat-square)
 ![GitHub Workflow Status](https://img.shields.io/github/workflow/status/TeamClimactic/Rango/deno?style=flat-square)
 
-Advance Modular Framework for Discordeno
+Advanced Modular Framework for Discordeno
```
Comment (AdityaTD) [inline README.md:10]: The current spelling is correct. however It's missing an article, aka "An" before "Advance".

**Comment by PlanetNoobGotHacked** [comment]
idk if this will cause any problems in the code

**Review by AdityaTD — CHANGES_REQUESTED** [review]

**Comment by PlanetNoobGotHacked** [comment]
ok i change

**Review by AdityaTD — CHANGES_REQUESTED** [review]
Incorrect. It's not "Advanced" it was right as is and was just missing an article.

**Comment by PlanetNoobGotHacked** [comment]
god fucking dammit

**Comment by PlanetNoobGotHacked** [comment]
i dont know english its not my first language this is bullying

**Comment by AdityaTD** [comment]
It's not my first language either :) Just going by grammar. If you're not comfortable with language-based PRs, just do programming.

**Comment by QuantumlyTangled** [comment]
This is not very poggers**.**

**Pull request closed without merging**